### DISCUSSION TITLE
Mercatox: DOGE Wallet Broken 1 Year!

### DISCUSSION THREAD

**Discussion by dfarrow125** [dogecoin/dogecoin#2786] [General]
The withdrawl wallet for Dogecoin has been broken on Mercatox exchange for almost 1 Year now.  Mercatox constantly promising they are working on it but to no avail.  As a large DOGE holder I have many other exchanges with DOGE thar fixed any of these issues as they came up.  Why isn't Mercatox fixing this?  I have around 250k doge on Mercatox and invested below 1 cent, before the popularity craze and sad to see early adopters get left behind bc of faulty exchanges while everyone else gets to moon.  Sure I have done well on my DOGE outside of Mercatox but it's the ethics here than got me spun.  How can we all moon and leave the Mercatox buyers in the dust.

Any plans or work from dogecoin on fixing Mercatox wallets?? We have been waiting a year now patiently.

**Comment by ReverseControl** [comment]
We only work on the Dogecoin Core Wallet. We have no affiliation to anything or anyone else. Raise any issues with them directly. @michilumin might want to chime in.

**Reply by dfarrow125** [reply]
Just figured that there is a relationship between Mercatox and DOGE bc u listed your coin there and it would mean something if they just took the wallet offline.  Going om a year now. Pretty sure they don't have the liquidity in your coin to pay back to what they sold so they just locked it up indefinitely since they basically stole the holding for themselves.  At this point there is t really any other explanation- I work for another alt coin and know it doesn't take a year to fix compatibility issues or updating soft fork code.  At this point I just figured I got ripped off and DOGE team is cool with that since they haven't tried to help directly with Mercatox or get a better explanation for those trading there.  Meanwhile users still trading it and it's basically fake.  🤦‍♂️

Fact are facts and actions speak louder than words, i.e. no action on any part 🤷‍♂️

**Reply by tealethetoolman** [reply]
Hey if you want to be the shibe who works with them to fix their implementation of the protocol, go for it. Thats why decentralization is so great. Otherwise, you just have to look at it as this exchange that nobody has heard of scammed you.

**Reply by dfarrow125** [reply]
No worries , used to be a huge supporter, had 3.5M DOGE, now I have almost none - weird this doesn't happen with any other open source stable I own 🥵

**Reply by tealethetoolman** [reply]
I’ve never had any problems with dogecoin, then again, I use reputable exchanges.

you should contact that exchange and find out where your coins are. As we always say, not your keys, not your coins. 

Also I’d get your lcc off there too ;)

**Reply by tealethetoolman** [reply]
I can’t believe I just did this legwork for you. 
[mercatox support](https://mercatox.com/contacts)

their website looks like it was made in ‘99. I definitely wouldn’t have had coins there.

**Comment by rnicoll** [comment]
We don't list on exchanges, or at a minimum this is not one we have requested to list Dogecoin. As with any open source software they are free to use it, we do not provide permission or request them to do so. That's why it's decentralised, because there's no centralised control.

**Reply by dfarrow125** [reply]
Yada Yada Yada ok

**Reply by dfarrow125** [reply]
And thanks for explaining defi 😆 I need that

Next can u break down this blockchain? Is it like a bunch of square cubes chained together ❤️ 

✌ tbh don't care anymore I'm out anyway
